lib/cartopy/examples/utm_all_zones.py

```diff
@@ -0,0 +1,47 @@
+"""
+Displaying all 60 zones of the UTM projection
+---------------------------------------------
+
+This example displays all 60 zones of the Universal Transverse Mercator
+projection next to each other in a figure.
+
+First we create a figure with 60 subplots in one row.
+Next we set the projection of each axis in the figure to a specific UTM zone.
+Then we add coastlines, gridlines and the number of the zone.
+Finally we add a supertitle and display the figure.
+"""
+__tags__ = ['Lines and polygons']
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+
+
+def main():
+    # Create a list of integers from 1 - 60
+    zones = range(1, 61)
+
+    # Create a figure and array of 60 subplot Axes objects
+    fig, axarr = plt.subplots(nrows=1,
+                              ncols=len(zones),
+                              figsize=(18, 6))
+
+    # Loop through each zone in the list
+    for zone in zones:
+
+        # Change the Axes object to a GeoAxes object with a specific UTM zone projection
+        axarr[zone - 1] = plt.subplot(1, len(zones), zone,
+                                      projection=ccrs.UTM(zone=zone, southern_hemisphere=True))
+
+        # Add coastlines, gridlines and zone number for the subplot
+        axarr[zone - 1].coastlines(resolution='110m')
```

Comment:
Did you take a look at adding land? How did it look?

Reply:
There are anomalies when plotting coastlines on in a number of the zones, which is made more obvious when land is added:
![utm_all_zones_land](https://user-images.githubusercontent.com/32906197/33062740-430243c2-ce98-11e7-8638-3d25ec43255e.png)

The anomalous zones are [2, 5, 8, 32, 35, 36, 38, 41, 52]:
![utm_anomalies_land](https://user-images.githubusercontent.com/32906197/33062760-55bccd66-ce98-11e7-9001-87cf6e629313.png)

So I chose to keep land out of the example until this has been fixed.

Reply:
Errors are returned when using `cartopy.feature.OCEAN` and `ax.stock_img()`, to the extent that nothing is displayed.